x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -795,7 +795,9 @@ export const FIELD = (readOnly?: boolean): FieldMap => ({
       return {
         'data-test-subj': 'syntheticsHeaderFieldRequestHeaders',
         readOnly,
-        contentMode: (requestBody as RequestBodyCheck).type,
+        contentMode: !!(requestBody as Record<string, string>)?.value
```

Comment:
Any reason why not use `RequestBodyCheck` here too?

```suggestion
        contentMode: !!(requestBody as RequestBodyCheck)?.value
```

Reply:
No reason, thanks.